test/e2e/adminapi_kubernetesobjects.go

```diff
@@ -26,7 +26,7 @@ var _ = Describe("[Admin API] Kubernetes objects action", func() {
 	const objName = "e2e-test-object"
 
 	When("in a standard openshift namespace", func() {
-		const namespace = "default"
```

Comment:
It looks like we are dropping support of the default manespace. I know that customers often deploy their workloads there (probably not intentionally in many cases).

Is there anything important for us to monitor in there? I don't think so.

@gvanderpotte and other RLs. Any thoughts?

Reply:
We don't deploy pods in default so IMO we should drop it.  It's a standard namespace in a sense that it exists in an openshift cluster, but besides that it's of no use to us.

Reply:
I'm unsure of what exactly we mean here when we say "dropping support"... but I know that generally, there are two very important objects that are expected to exist in `default` from a Kubernetes perspective, and we see that in OpenShift/ARO clusters too:

- `service/kubernetes` - a ClusterIP Service used by Pods to hit the K8s API
- `service/openshift` - apparently just an ExternalName record to `service/kubernetes`...

`service/kubernetes` is actually so important, that if you delete it the API automatically recreate it because it's how all controllers interact with the cluster API.

Reply:
@SudoBrendan - Thanks for your comment. So the problem is what exactly the namespaces we really care about. If we only care about the namespaces that the customer deploy their workloads to, then we probably don't have to keep the `default` and the other 3 initial namespaces as below: 
- `kube-system`
- `kube-public`
- `kube-node-lease`

However, if we want to get monitoring from the kubernetes system level for auditing purpose (Checking the API traffic that the pods sending requests to the Kubernetes master) Then, it makes sense to keep the `default` and the `kube-` namespaces. Also, the previous function `IsOpenShift` and `IsOpenShiftSystemNamespace` also handled the `default` and `kube-` namespaces. Our new filtering function should support the same as before ? How do you think @bennerv @m1kola ?

Reply:
I think there is some confusion in this thread.

We collect the following things:

* Logs (audit logs and logs from OCP workloads). It is a separate config within the cluster.
    Here is where we define namespaces from which we gather logs:
    https://github.com/Azure/ARO-RP/blob/6c9de0ae5c672e6c4323d0ca5f7e55dbeeb5d124/pkg/operator/controllers/genevalogging/const.go#L73-L76
    [This file](https://github.com/Azure/ARO-RP/blob/6c9de0ae5c672e6c4323d0ca5f7e55dbeeb5d124/pkg/operator/controllers/genevalogging/const.go) contains other log related configs (including audit logs).
    **Note:** this PR does not affect logs.
* Metrics. These are gathered by monitor component (`pkg/monitor`). This is what we are changing in this PR.

---

I do not think we need `default`, `kube-*` namespaces here. We do not deploy any OCP things we gather metrics on into these namespaces as far as I know. But customers can deploy pods, jobs, etc into these namespaces and it will introduce noise into our metrics.